itm/itm_scenario_runner.py

```diff
@@ -8,6 +8,10 @@
 def get_swagger_class_enum_values(klass):
     return [getattr(klass,i) for i in dir(klass) if not i.startswith("_") and isinstance(getattr(klass,i), str)]
 
+# A TA2 performer would probably want to get alignment target ids from configuration or command-line.
+SOARTECH_ALIGNMENT = 'kdma-alignment-target-1'
+ADEPT_ALIGNMENT = 'ADEPT-alignment-target-1-eval'
```

Comment:
I feel like we should flag this for discussion with TA1/TA2 but not let it hold up checking this in.

Reply:
Yeah, I'm not sure exactly how these IDs are communicated between teams let alone between software components.  Right now they're just listed in the TA1 repo READMEs.  These lines simply mean that the ADMs will need to get the ids from somewhere.